apps/condo/pages/tour/guide.tsx

```diff
@@ -188,21 +188,22 @@ const AboutAppBlock = () => {
                                     }
                                 </Col>
                                 {
-                                    get(guideModalCardLink, [locale, type]) && (
+                                    locale === 'ru' && modalCardExamples[type] && (
```

Comment:
why `ru` only?

Reply:
Yep, best to make smth like

modal_card_examples_lalala: { en: ... ru: ... }